src/status_im/subs/communities.cljs

```diff
@@ -55,6 +56,37 @@
  (fn [[{:keys [community-id]} communities]]
    (get-in communities [community-id :members])))
 
+(re-frame/reg-sub
+ :communities/current-channel-contacts
+ :<- [:communities]
+ :<- [:contacts/contacts]
+ :<- [:profile/profile]
+ (fn [[communities contacts {:keys [public-key preferred-name name display-name] :as current-account}]
+      [_ community-id channel-id]]
+   (let [current-contact (some->
+                           current-account
```

Comment:
maybe it's better to split this into a few smaller subs and re-use those?
e.g current-contact, all-contacts and then you just need this sub to handle the online/offline status.

Also to that point, I think the name should be more descriptive to that.
e.g :communities/current-channel-contacts-status or -visibility-status